src/storageClients/CassandraTableStorageManager.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Promise = require('bluebird');
+
+const BATCH_LIMIT = 10;
+
+module.exports = {
+
+  prepareInsertTopic: function(topic) { //TODO: the insert statement to reflect the final topic schema
+    return {
+      query: 'INSERT INTO topics (id, topic, value) VALUES (?, ?, ?)',
+      params: [ topic.id, topic.topic, topic.value ]
+    };
+  },
+
+  batch: function(client, queries){
+    return new Promise((resolve, reject) => {
+      Promise.all(getBatchPromises(client, queries))
+        .then(res => {
+          resolve(res);
+        })
+        .catch(err => {
+          reject(err);
+        });
+    });
+  }
+
+};
+
+let getBatchPromises = (client, queries) => {
+  let insertPromises = [];
+  for ( let i = 0, j = queries.length; i < j; i += BATCH_LIMIT ) {
```

Comment:
Is this iterating over the array in chunks? If so: would it be more readable to extract a helper like in this StackOverflow [post](https://stackoverflow.com/a/10456644/3817588)?